### PR TITLE
Length

### LINKED ISSUE #642: Fix length
There is currently a method called `length` that lives in `/lib/length.js`.
It is incomplete and needs to be fixed!

It should return the length of a list, which can be an array or a string in this case.

### Acceptance Criteria:
1. Running `yarn test <methodName>` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by kenhkelly**
Closes #642 

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: test/length.test.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ describe("length", () => {
   });
 
   test("length of 'length' to be 5", () => {
```
Comment (ktilcu) [inline test/length.test.js]: mind fixing this line too?

**Comment by kenhkelly** [comment]
@ktilcu I updated the test string

**Comment by ktilcu** [comment]
@kenhkelly Thanks! Though, I think this is the perimeter of a triangle. If you wrap that up we can ge tit done.
